### DISCUSSION TITLE
Ratings in store

### DISCUSSION THREAD

**Discussion by Manu99it** [ChristopherKlay/StadiaEnhanced#28] [Ideas & Requests]
I like the button to search a game on Metacritic. But what about displaying directly in the game's page the rating? [Stadiagamedb](https://github.com/nilicule/StadiaGameDB) has them, you could use that

**Comment by ChristopherKlay** [comment]
The issue with that is that Metacrritic currently doesn't actually feature a rating for games on Stadia (outside of 2 games) and pulling ratings from other platforms isn't making all that much sense, when the rating is also based on how well the game works; which drastically changes with the quality of the Stadia port (i.e. The Crew 2 at 65-70 would most likely be below 60 on Stadia).

Once Stadia grows (and actually gets it's own category on Metacritic to filter for) i will definitely display that and also link to the game itself on Metacritic, instead of just a search.

**Comment by Manu99it** [comment]
Makes sense, thank you!
